spec/views/templates/search/search_home_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'rails_helper'
+
+RSpec.describe Templates::Search::SearchHome, type: false do
+  it { is_expected.to be_a(ApplicationView) }
+
+  describe '#test' do
+    it 'says hello' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.